framework/py/flwr/common/inflatable_test.py

```diff
@@ -68,6 +77,16 @@ def test_deflate_and_inflate() -> None:
     obj_ = CustomDataClass.inflate(obj_b)
     assert obj_.data == obj.data
 
+    # Assert
+    # Inflate passing children raises ValueError
+    with pytest.raises(ValueError):
+        CustomDataClass.inflate(
+            obj_b,
+            children=[
+                obj,
+            ],
+        )
```

Comment:
Just out of curiosity, why the code is put in 6 lines?

```suggestion
        CustomDataClass.inflate(obj_b, children=[obj])
```

Reply:
formatting script did it 😅

Reply:
done